lib/pages/symptoms.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:flutter/material.dart';
+
+class Symptoms extends StatefulWidget {
+  @override
+  _SymptomsState createState() => _SymptomsState();
+}
+
+class _SymptomsState extends State<Symptoms> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: Text('Symptoms'),
+        centerTitle: true,
+      ),
+      body: Center(
+        child: Column(
+          mainAxisAlignment: MainAxisAlignment.center,
+          children: <Widget>[
+            Text(
+              'Symptoms',
+            ),
```

Comment:
Add this style   
`style: TextStyle(
                  color: Colors.red[300],
                  fontWeight: FontWeight.bold,
                  fontSize: 20.0,),
                  textAlign: TextAlign.center,
            )`